nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/main/java/org/apache/nifi/csv/CSVReader.java

```diff
@@ -79,6 +79,18 @@ public class CSVReader extends SchemaRegistryService implements RecordReaderFact
             .required(true)
             .build();
 
+    public static final PropertyDescriptor TRIM_DOUBLE_QUOTE = new PropertyDescriptor.Builder()
+            .name("Trim double quote")
+            .description("Whether or not to trim starting and ending double quotes. For example: with trim string '\"test\"'"
+                    +" would be parsed to 'test', without trim would be parsed to '\"test\"'."
+                    + "If set to 'false' it means full compliance with RFC-4180. Default value is true, with trim.")
+            .expressionLanguageSupported(ExpressionLanguageScope.NONE)
+            .allowableValues("true", "false")
+            .defaultValue("true")
+            .dependsOn(CSVUtils.CSV_FORMAT, CSVUtils.RFC_4180)
```

Comment:
Please add validator and displya name, something like:
```
.name("trim-double-quotes")
.displayName("Trim double quotes")
.addValidator(StandardValidators.BOOLEAN_VALIDATOR)
```

Otherwise it looks good and after this change I will merge it

Reply:
Thanks @simonbence, validator added, please merge it